etl/churn.kp/knowledge.md

```diff
@@ -496,7 +486,7 @@ def compute_churn_week(df, week_start, bucket, prefix):
                 x.get('start_version', 'unknown'),
                 x.get('sync_usage', 'unknown'),
                 x.get('current_version', 'unknown'),
-                x.get('current_week', 'unknown'),
+                x.get('current_week', -1),
```

Comment:
Sure, and I was initially thinking of leaving it null. But it seems inconsistent with how the rest of values have defaults.